jobs/pacman-cloud-discovery/src/main/java/com/tmobile/cso/pacman/inventory/InventoryCollectionJob.java

```diff
@@ -61,8 +61,10 @@ public static Map<String, Object> execute(Map<String,String> params){
 		try {
 			ConfigUtil.setConfigProperties();
 		} catch (Exception e) {
-			log.error("Error fetching config", e);
+			//below logger is used to create data alert
+			log.error("Exception occurred in job aws-data-collector-job");
 			ErrorManageUtil.uploadError("all", "all", "all", "Error fetching config "+ e.getMessage());
+			ErrorManageUtil.jobStatus = "failed";
```

Comment:
from my understanding this `ErrorManageUtil` is just static utility class. and here you changing something inside it... Utility class must not have any state whatsoever. If you need to pass/save job status - you need to find another way to do that.

Reply:
moved the jobStatus attribute to a different class.